.template/variants/web/app/template.rb

```diff
@@ -3,13 +3,20 @@
 # Javascript
 directory 'app/javascript'
 
-if File.exist?('app/views/layouts/application.html.erb')
-  insert_into_file 'app/views/layouts/application.html.erb', before: %r{</head>} do
+erb_layout_file = 'app/views/layouts/application.html.erb'
+slim_layout_file = 'app/views/layouts/application.html.slim'
+
+erb_layout_exists = File.exist?(erb_layout_file)
+slim_layout_exists = File.exist?(slim_layout_file)
+
+if erb_layout_exists
+  insert_into_file erb_layout_file, before: %r{</head>} do
     <<~ERB.indent(2)
       <%= javascript_include_tag "application", "data-turbo-track": "reload", defer: true %>
     ERB
   end
-else
+# The slim layout (CRUD addon) already has the javascript include tag
+elsif !slim_layout_exists
```

Comment:
If I understand it correctly 💭 

```suggestion
elsif slim_layout_exists
```

Reply:
This one was confusing. It meant that we throw the error if the layout is neither ERB neither Slim.

Renamed in 0243e3d, let me know if that's clearer or not 😇